tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java

```diff
@@ -0,0 +1,250 @@
+            String endTime = element.getAsJsonObject().get("end_time").getAsString();
+            Date startTimeDate = parseISO8601Date(startTime);
+            Date endTimeDate = parseISO8601Date(endTime);
+            times.add(new TimeStatisticForOneTask(startTimeDate, endTimeDate, element.getAsJsonObject().get("name").getAsString()));
+        }
+        this.timesForEachTask = times;
+    }
+
+
+    private Long getTotalTimeInMilliseconds() {
+        if (timesForEachTask == null) {
+            setTimeForEachTask();
+        }
+        Long totalTimeTaken = 0L;
+        for (TimeStatisticForOneTask time: timesForEachTask) {
+            totalTimeTaken += time.getTimeTakenInMilliseconds();
+        }
+        return totalTimeTaken;
+    }
+
+
+    private void printTimeStatistic() {
+        if (timesForEachTask == null) {
+            setTimeForEachTask();
+        }
+        for (TimeStatisticForOneTask time: timesForEachTask) {
+            Long timeTakenInMilliseconds = time.getTimeTakenInMilliseconds();
+            Long milliseconds = timeTakenInMilliseconds % MILLISECONDS_IN_SECOND;
+            Long seconds = timeTakenInMilliseconds / MILLISECONDS_IN_SECOND;
+            Long minutes = seconds / SECONDS_IN_MINUTE;
+            seconds = seconds % SECONDS_IN_MINUTE;
+
+            out("");
+            out("TASK NAME: " + time.getTaskName());
+            out("START TIME: " + time.getStartTime().toString());
+            out("END TIME: " + time.getEndTime().toString());
+            out("DURATION: " + minutes + " minutes " + seconds + " seconds " + milliseconds + " milliseconds");
+        }
+        out("");
+        Long totalTimeTakenMilliseconds = getTotalTimeInMilliseconds();
+        Long milliseconds = totalTimeTakenMilliseconds % MILLISECONDS_IN_SECOND;
+        Long seconds = totalTimeTakenMilliseconds / MILLISECONDS_IN_SECOND;
+        Long minutes = seconds / SECONDS_IN_MINUTE;
+        seconds = seconds % SECONDS_IN_MINUTE;
+        out("TOTAL TIME: " + minutes + " minutes " + seconds + " seconds " + milliseconds + " milliseconds");
+    }
+
+    public void printRunStatistics() {
+        if (isWorkflowFinished()) {
+            out("RUN STATISTICS:");
+            out("ENTRY NAME: " + getCompleteEntryName());
+            out("END STATE: " + state);
+            if (state.equals("COMPLETE")) {
+                printTimeStatistic();
+            }
+        } else {
+            errorMessage("Workflow is not finished, statistic are not available yet", COMMAND_ERROR);
+        }
+    }
+
+    private String getTotalTimeInISO861Standard() {
+        if (timesForEachTask == null) {
+            setTimeForEachTask();
+        }
+        Long totalTimeMilliseconds = getTotalTimeInMilliseconds();
+        Duration duration = Duration.ofMillis(totalTimeMilliseconds);
+        return duration.toString();
+    }
+
+    private Execution.ExecutionStatusEnum getExecutionStatus() {
+        if ("COMPLETE".equals(state)) {
```

Comment:
I would make `"COMPLETE"` a constant since it's used more than once

Reply:
Fixed, thanks!